src/landpatterns/BGA/planner.stanza

```diff
@@ -25,32 +25,57 @@ This type is derived from two parent implementations:
 and how they should be constructed
 2.  {@link type:GridPlanner} which manages the placement of the created pads
 at particular grid locations.
+@member pad-config Defines the features of each individual BGA pad.
+@member pose Pose of the landpattern.
 <DOC>
 public defstruct BGA-PadPlanner <: PadPlanner & GridPlanner:
   doc: \<DOC>
-  Defines the features of each individual BGA pad.
+  @title pad-config
+  @desc Defines the features of each individual BGA pad.
   This defines the shape and parameters for the copper,
   soldermask opening, paste application, etc.
   <DOC>
   pad-config:PadConfig with:
     default => DEF-PAD-CONFIG
 
   ; GridPlanner Base
+  doc: \<DOC>
+  @title pose
+  @desc Pose of the landpattern (default is loc(0.0, 0.0))
```

Comment:
This is the one place where this is interesting and adds value. The default argument for both defstruct fields and function arguments is not an easy thing for us to extract from the definitions database. This might be very useful for extracting the default values into the comments and keeping them up to date.